packages/cli/src/commonlib/cacheAccess.ts

```diff
@@ -201,3 +202,60 @@ export async function loadAccountId(accountName: string) {
 
   return undefined;
 }
+
+export class AzureSpCrypto {
+  private static readonly accountCrypto: AccountCrypto = new AccountCrypto("sp");
```

Comment:
Just use "azure" as account name to keep consistency with user azure account. No need to add another crypto key for SP.